src/ddmd/backend/dwarf.d

```diff
@@ -1,4 +1,4 @@
-/* Dwarf debug
+/** Dwarf debug
  *
  * Source: $(DMDSRC backend/_dwarf.d)
  */
```

Comment:
> I know that it's a debug file

It's for generation of [DWARF debug information](https://en.wikipedia.org/wiki/DWARF), so it's definitely not a utility file - it needs to be part of the documentation.